src/Raven.Studio/wwwroot/App/views/database/query/query.html

```diff
@@ -216,36 +222,42 @@ <h2 class="on-base-background">
                 <div class="panel-body absolute-fill js-time-series-graph-container" data-bind="if: showTimeSeriesGraph(), visible: showTimeSeriesGraph()">
                     <div class="absolute-fill flex-vertical" data-bind="compose: $root.currentTab"></div>
                 </div>
+                <div class="panel-body absolute-fill js-time-series-graph-container" data-bind="if: showMapView(), visible: showMapView()">
```

Comment:
js-time-series-graph-container" data-bind="if: showMapView()

time series and map view?

Reply:
removed that ts class